apps/spotlight/package.json

```diff
@@ -6,7 +6,7 @@
     "astro": "astro",
     "build": "astro build",
     "clean": "rm -rf dist",
-    "dev": "astro dev",
+    "dev": "astro dev --host",
```

Comment:
IIRC `pnpm` will also proxy args, so you could do `pnpm dev --host` in `Dockerfile.e2e`.